website/docs/migration/premigration-testing.md

```diff
@@ -83,8 +100,8 @@ You would do the following:
 Continuing with the same example above, we can further examine the diff output while in `diff-proxy` mode.
 
 1. Run FerretDB in `diff-proxy` mode.
-   This can again be achieved by using the `--mode` [flag](../configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable as already shown.
-2. Re-run the query.
+   This can again be achieved by using the `--mode` [flag](../configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable to `diff-proxy`.
+2. Follow the same instructions as the one for `diff-normal` above to run FerretDB in `diff-proxy` mode and re-run the query.
```

Comment:
Is diff output is the same regardless of running `diff-proxy` or `diff-normal`? It's a bit strange to have diff output under this header but not under previous `diff-normal` section.